molvis/molvisbase/src/util/molvisutils.cpp

```diff
@@ -186,8 +190,18 @@ std::vector<Bond> computeCovalentBonds(const Atoms& atoms) {
                                       atoms.atomicNumbers[atom2], atoms.positions[atom2]);
     };
 
+    auto isNobleGas = [](Element symbol) {
```

Comment:
Move to location of isMetallic?

Reply:
Discussed at length and we concluded that noble gases are hardly used anywhere else. Hence it was only included here.